core/build/src/lib.rs

```diff
@@ -2,10 +2,57 @@
 
 #![doc = include_str!("../README.md")]
 
+use bindgen::{callbacks::ParseCallbacks, Builder, EnumVariation};
 use std::{env, path::PathBuf};
 
 static DEFAULT_SGX_SDK_PATH: &str = "/opt/intel/sgxsdk";
 
+/// Provides a default [bindgen::callbacks::ParserCallbacks::item_name]
+/// implementation that works with most SGX types.
+///
+/// # Arguments
+/// * `name` - The name of the type to determine the bindgen name of.
+pub fn sgx_item_name(name: &str) -> Option<String> {
```

Comment:
Nit: Perhaps rename this function to something more descriptive / uses a verb? I'm not sure what it does by just looking at it. Would `add_sgx_item` be an acceptable name?

Reply:
I went with `sgx_normalize_item_name()`.  I think that better describes what's going on.

Reply:
all though the "sgx" prefix may be my C namespace mentality coming in, so feel free to suggest a better name there